package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "proget-universal-bower-client",
-  "version": "0.3.1",
```

Comment:
I was asking myself the same question before increasing the minor number and not the patch.

Both have their point:

Patch: 
 - It's a small change

Minor:
- It's a new feature
- There was no bug fix.

Maybe an argument for both:
- It's a new way to use an already existing command.

I initially don't name that branch a version number because of that.
What are you thinking about that @fvgoulet ?